### ISSUE TITLE
Plot bookkeeping

### ISSUE THREAD

**Issue by aimalz** [nataliastylianou/Redshifts#2]
These are just some notes of to-do items from our recent call:
- [ ] Add axis labels to plots
- [ ] Add titles in plots including names of test set and training set
- [ ] Try removing `.astype(int)` in metrics (e.g. [https://github.com/nataliastylianou/Redshifts/blob/main/samples_photoz.py#L137](Fig 3.))
- [ ] For faster rendering of plots with many points (e.g. [https://github.com/nataliastylianou/Redshifts/blob/main/samples_photoz.py#L122](Fig. 2)), try plotting with `plt.hist2d` and/or`alpha=` keyword (optional)
